### ISSUE TITLE
Tooltip Signals Containing Plain String Values are Converted to Arrays

### ISSUE THREAD

**Issue by PBI-David** [deneb-viz/deneb#201]
When using Vega Editor to display tooltips, they are rendered correctly. However, the exact same code in Deneb produces tooltips that look like they are turned into some kind of array as below.

![image](https://user-images.githubusercontent.com/86846675/152964729-e55fceb2-e2fb-441e-9098-7a3e024c92f6.png)

.pbix attached for investigation.
[Tree Test2.zip](https://github.com/deneb-viz/deneb/files/8022742/Tree.Test2.zip)

Using the following works fine:

```
"tooltip": {
            "signal": "datum"
          }
```
 
But using below produces the weird result. Using field instead of signal does the same and interestingly trying to access a number instead of text (i.e. datum.size) produces an empty tooltip.

```
     "tooltip": {
            "signal": "datum.name"
          }
```

I'm probably missing something obvious so appreciate any pointers.

**Comment by dm-p** [comment]
Looks like it could be an issue with how we process the signal into Power BI's `VisualTooltipDataItem` elements if a single datum field is provided. most of our focus in getting the integration working was for Vega-Lite, so that may be something specific to deal with here. I'll try to find some time to explore further and see if there's anything we need to take into account.

As a workaround, the following approach from the [map Vega tooltip example](https://vega.github.io/vega-tooltip/vega-examples.html) should help: specify a valid object of key/value pairs inside the `signal` expression, e.g.:

```
"tooltip": {
    "signal": "{'Name': datum.name, 'ID': datum.id}"
}
```

In your example specification, this now looks as follows when a mark is hovered-over:

![image](https://user-images.githubusercontent.com/10572054/153329045-54fb80c4-cc92-46c2-82db-90fd35b4be9d.png)

**Comment by dm-p** [comment]
Just some notes, while I have the debugger running - turning off the Power BI tooltip handler and falling back to Vega's correctly shows `datum.name` as a string, e.g.:

![image](https://user-images.githubusercontent.com/10572054/153333857-492b3cc4-1776-46e3-805d-f238171fdc07.png)

In this case, the signal's `tooltip` property is a plain string, rather than an iterable:

![image](https://user-images.githubusercontent.com/10572054/153334072-05c73335-886e-4741-9fae-fe22176467f3.png)

As we normally assume an object of key/value pairs (because this is how Power BI tooltips need to be structured), this will indeed get split into an array for each letter.

**Comment by dm-p** [comment]
This will be fixed in the next 1.2 alpha build, which will trigger on the next pull request. 

Because Power BI tooltips need key/value pairs as part of their definitions, if a tooltip signal contains a scalar value rather than an object, we will reconstruct it with a blank key and use the scalar value as the corresponding property value in order to game the Power BI tooltip handler into similar behaviour as the inbuilt vega tooltip, e.g.:

![image](https://user-images.githubusercontent.com/10572054/156285237-0eef9b2e-25ba-40d7-b5f8-e7d7fb35485a.png)
